src/wui/wui_api.h

```diff
@@ -9,17 +9,45 @@
 #ifndef _WUI_API_H_
 #define _WUI_API_H_
 
+#include "lwip/netif.h"
+#include "eeprom.h"
```

Comment:
remove this, crossed dependencies and it's used only for string sizes which should be defined here and included into eeprom

Reply:
it should be opposit. api function is not to include to EEPROM